test/rpc/conftest.py

```diff
@@ -57,8 +58,13 @@ def profiles_root(tmpdir):
 def project_root(tmpdir):
     return tmpdir.mkdir('project')
 
-
 def postgres_profile_data(unique_schema):
+    database = 'database'
+    if sys.platform == 'win32':
+        datbase = 'localhost'
```

Comment:
I think this is supposed to be 'database'

Reply:
ah ya, but who develops on windows anyway 🥴

Reply:
True :-)